jmbo_calendar/views.py

```diff
@@ -1,6 +1,4 @@
 from django.utils.translation import ugettext as _
-from django.core.urlresolvers import reverse
```

Comment:
from jmbo import USE_GIS

Please use that variable in checks, not ImportError directly.